tests/FSharp.Core.UnitTests/FSharp.Core.UnitTests.fsproj

```diff
@@ -3,8 +3,8 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net8.0;net472</TargetFrameworks>
-    <TargetFrameworks Condition="'$(OS)' == 'Unix'">net8.0</TargetFrameworks>
+    <TargetFrameworks>$(FSharpNetCoreProductTargetFramework);net472</TargetFrameworks>
```

Comment:
Why not create a variable for NET472 as well then? We now have it in scripts so it would be uniform.

Reply:
Two reasons
1. It's outside of what is getting fixed here (source build).
2. We want to eventually get rid of it, I don't want to waste time on it.

Reply:
If you unify this at some point, consider using one of these variables: https://github.com/dotnet/arcade/blob/26e06f1515561b634abf0f9e511f29d1baf56be5/src/Microsoft.DotNet.Arcade.Sdk/tools/TargetFrameworkDefaults.props#L24-L26